q2_annotate/plugin_setup.py

```diff
@@ -816,24 +814,21 @@
 plugin.methods.register_function(
     function=q2_annotate.eggnog._eggnog_hmmer_search,
     inputs={
-        'sequences':
+        'seqs':
             SampleData[Contigs] | SampleData[MAGs] | FeatureData[MAG],
         'idmap': EggnogHmmerIdmap,
-        'pressed_hmm_db': ProfileHMM[PressedProtein],
+        'db': ProfileHMM[PressedProtein],
         'seed_alignments': GenomeData[Proteins]
     },
     parameters={
         'num_cpus': Int,
         'db_in_memory': Bool,
     },
     input_descriptions={
-        'sequences': 'Sequence data of the contigs we want to '
-                     'search for hits.',
-        'idmap': 'List of protein families in `hmm_db`.',
-        'pressed_hmm_db': 'Collection of Profile HMMs in binary format '
-                          'and indexed.',
-        'seed_alignments': 'Seed alignments for the protein families in '
-                          '`hmm_db`.'
+        'seqs': 'Sequence data of the contigs we want to search for hits.',
```

Comment:
Could you please update this description to say "sequences" instead of "contigs"? I think we forgot to change when adding more input types.